packages/react-next/src/components/TextField/TextField.types.ts

```diff
@@ -51,6 +51,12 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   componentRef?: IRefObject<ITextField>;
 
+  /**
+   *  Optional callback to access the root DOM element.
+   *  @deprecated Temporary solution which will be replaced with ref in the V8 release.
```

Comment:
Especially for `@deprecated` tags it's important to only have one leading space in comments--otherwise the lint rule won't pick them up properly.
```suggestion
   * Optional callback to access the root DOM element.
   * @deprecated Temporary solution which will be replaced with ref once TextField is converted to a function component.
```